api/v1beta1/frrconfiguration_types.go

```diff
@@ -80,6 +80,16 @@ type Router struct {
 	// Prefixes is the list of prefixes we want to advertise from this router instance.
 	// +optional
 	Prefixes []string `json:"prefixes,omitempty"`
+
+	// Imports is the list of imports we want for this router / vrf
+	// +optional
+	Imports []Import `json:"imports,omitempty"`
```

Comment:
because frr allows more complex imports than that, and I wanted to leave the door open in case we want to extend. If we stick with []string, we'll have to another field even if it semantically belongs to the imports.

Reply:
Could you name for my knowledge which complex imports you have in mind? I am in favor comment the reasoning why not `[]string`

Reply:
it's the import from l2vpn logic frr provides. I just don't want to prevent that and make the api extensible.

Reply:
https://docs.frrouting.org/en/latest/bgp.html#clicmd-rt-vpn-import-export-both-RTLIST...